composer.json

```
@@ -45,7 +45,7 @@
         "doctrine/collections": "^1.0",
         "doctrine/common": "^2.13 || ^3.0",
         "doctrine/event-manager": "^1.0",
-        "doctrine/persistence": "^1.3.3 || ^2.0",
+        "doctrine/persistence": "^2.0",
```

```suggestion
        "doctrine/persistence": "^2.2",
```
Since it's the version when the key was changed